test/envs/regional-dr-argocd.yaml

```diff
@@ -0,0 +1,43 @@
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+# Environment for testing Regional-DR.
+---
+name: "rdr"
+
+ramen:
+  hub: hub
+  clusters: [dr1, dr2]
+  topology: regional-dr
+  features:
+    volsync: true
+
+templates:
+  - name: "dr-cluster"
+    driver: "$vm"
+    container_runtime: containerd
+    network: "$network"
+    cpus: 4
+    memory: "6g"
+    extra_disks: 1
+    disk_size: "50g"
+    feature_gates:
+      - StatefulSetAutoDeletePVC=true
+  - name: "hub-cluster"
+    driver: "$vm"
+    container_runtime: containerd
+    network: "$network"
+    cpus: 2
+    memory: "4g"
+    workers:
+      - addons:
+          - name: argocd
+            args: ["hub", "rdr-dr1", "rdr-dr2"]
```

Comment:
This will never work - there is no clusters "rdr-dr1" and "rdr-dr2". This must use the same names as the profile names ("dr1", "dr2").

Reply:
yes, this is a just a trial, I already changed back.